src/modules/loginForm/LoginForm.tsx

```diff
@@ -0,0 +1,89 @@
+import {
+  Box,
+  Button,
+  Checkbox,
+  Container,
+  Flex,
+  FormControl,
+  FormErrorMessage,
+  FormLabel,
+  Heading,
+  Input,
+  Link,
+  Stack,
+} from "@chakra-ui/react";
+import React, { ReactElement } from "react";
+import { useForm } from "react-hook-form";
+
+type FormData = {
+  username: string;
+  password: string;
+};
+export default function LoginForm(): ReactElement {
+  const { register, setValue, handleSubmit, errors } = useForm<FormData>();
+  const onSubmit = handleSubmit((data) => {
+    console.log("🚀 ~ file: LoginForm.tsx ~ line 32 ~ onSubmit ~ data", data);
+  });
+  return (
+    <Container>
+      <Box as="form" w="360px" p={4} onSubmit={onSubmit}>
+        <Heading paddingBottom={6}>Sign In</Heading>
+        <FormControl
+          paddingBottom={4}
+          id="username"
+          isRequired
+          isInvalid={!!errors.username}
```

Comment:
Can get invalid value from `fromState`. Read this for more info https://react-hook-form.com/api/#formState